### PR TITLE
Add input validation for FT handler

### PR DESCRIPTION
**Pull request by margaretqian**

### CODE REVIEW

File: examples/inference-deployments/mpt/mpt_7b_ft_handler.py [hunk 1]
```diff
@@ -123,6 +123,12 @@ class MPTFTModelHandler:
         'random_seed': True
     }
 
+    HF_TO_FT_KWARGS_MAPPING = {
+        'max_new_tokens': 'output_len',
+        'num_beams': 'beam_width',
+        'do_sample': None
```
[the remark this answers is not in the document]
Comment (dskhudia) [reply examples/inference-deployments/mpt/mpt_7b_ft_handler.py:129]: Just found out that setting top_k = 1 for FT turns sampling off.

File: examples/inference-deployments/mpt/mpt_7b_ft_handler.py [hunk 2]
```diff
@@ -123,6 +123,12 @@ class MPTFTModelHandler:
         'random_seed': True
     }
 
+    HF_TO_FT_KWARGS_MAPPING = {
+        'max_new_tokens': 'output_len',
```
[the remark this answers is not in the document]
Comment (dskhudia) [reply examples/inference-deployments/mpt/mpt_7b_ft_handler.py:127]: max_length is prompt + output_len; output_len should map to max_new_tokens

File: examples/inference-deployments/mpt/mpt_7b_ft_handler.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (margaretqian) [reply examples/inference-deployments/mpt/mpt_7b_ft_handler.py:129]: added special case message!

File: examples/inference-deployments/mpt/mpt_7b_ft_handler.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dakinggg) [reply examples/inference-deployments/mpt/mpt_7b_ft_handler.py:127]: sounds like we should remove the aliasing of `max_length` and `output_len` @margaretqian ?

**Pull request closed without merging**